Work around incorrect -Wmaybe-uninitialized warning due to this GCC bug:

  https://gcc.gnu.org/bugzilla/show_bug.cgi?id=69537

Index: glibc-2.22-709-g8a71d2e/nis/ypclnt.c
===================================================================
--- glibc-2.22-709-g8a71d2e.orig/nis/ypclnt.c
+++ glibc-2.22-709-g8a71d2e/nis/ypclnt.c
@@ -584,7 +584,7 @@ yp_master (const char *indomain, const c
 {
   ypreq_nokey req;
   ypresp_master resp;
-  enum clnt_stat result;
+  int result;
 
   if (indomain == NULL || indomain[0] == '\0' ||
       inmap == NULL || inmap[0] == '\0')
